chainio/clients/avsregistry/writer_test.go

```diff
@@ -837,6 +837,36 @@ func TestCreateAVSRewardsSubmission(t *testing.T) {
 	require.Equal(t, gethtypes.ReceiptStatusSuccessful, receipt.Status)
 }
 
+func TestAddStrategies(t *testing.T) {
+	clients, _ := testclients.BuildTestClients(t)
+	chainWriter := clients.AvsRegistryChainWriter
+	chainReader := clients.AvsRegistryChainReader
+
+	// contractAddrs.Erc20MockStrategy is already set as a strategy at index 0
+	strategyParam := stakeregistry.IStakeRegistryTypesStrategyParams{
+		Strategy:   gethcommon.HexToAddress(testutils.ANVIL_THIRD_ADDRESS),
```

Comment:
Do we need it to be the third anvil address? If not, can we use a random one like `0x1` or similar? That way any reader knows its value is not important